src/features/view-bids/ViewBids.tsx

```diff
@@ -1,17 +1,144 @@
 import { FC } from 'react';
 
 import { useWeb3 } from '../../lib/hooks/useWeb3';
+import { useViewBidsData } from './useViewBidsData';
+import { sortBidsByTime } from '../../lib/util/bids/bids';
+import { formatEthers } from '../../lib/util';
+import { truncateAddress } from '@zero-tech/zapp-utils/formatting/addresses';
+import { Bid } from '@zero-tech/zauction-sdk';
+import moment from 'moment';
 
-import { ConnectWallet } from '../ui/ConnectWallet';
+import {
+	AcceptBidButton,
+	AcceptBidButtonProps,
+} from '../../features/accept-bid';
+import { TextStack, TextStackProps, Wizard } from '@zero-tech/zui/components';
 
-export const ViewBids: FC = () => {
+import styles from './ViewBids.module.scss';
+
+export interface ViewBidsProps {
+	zna: BidListProps['zna'];
+}
+
+export const ViewBids: FC<ViewBidsProps> = ({ zna }) => {
 	const { account } = useWeb3();
+	const { bids, isLoadingBids, owner, paymentTokenSymbol } =
+		useViewBidsData(zna);
+
+	const sortedBids = sortBidsByTime(bids);
+
+	const isOwnedByUser: BidListProps['isAcceptBidEnabled'] =
+		owner?.toLowerCase() === account?.toLowerCase();
+
+	const bidsToShow: BidListProps['bids'] = isOwnedByUser
+		? sortedBids.filter(
+				(bid) => bid.bidder.toLowerCase() !== account?.toLowerCase(),
+		  )
+		: sortedBids;
+
+	return (
+		<Wizard.Container className={styles.Container}>
+			<Header />
+
+			{!isLoadingBids ? (
```

Comment:
Solution to display loading bid data in the ui needs to be handled prior to mapping in `BidList`. Using skeleton text available in `TextStack` won't be visible due to the order bid data is passed down. 
Solution result:

https://user-images.githubusercontent.com/39112648/199738637-8ca40f4d-effb-4b0e-95c3-20af8885d58d.mov